src/pymor/algorithms/ml/nn/neural_networks.py

```diff
@@ -0,0 +1,65 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+from pymor.core.config import config
+
+config.require('TORCH')
+
+import torch
+import torch.nn as nn
+
+from pymor.core.base import BasicObject
+
+
+class FullyConnectedNN(nn.Module, BasicObject):
+    """Class for neural networks with fully connected layers.
+
+    This class implements neural networks consisting of linear and fully connected layers.
+    Furthermore, the same activation function is used between each layer, except for the
+    last one where no activation function is applied.
+
+    Parameters
+    ----------
+    layer_sizes
```

Comment:
Rename `layer_sizes` to `hidden_layers` (or vice versa) and add `input_dimension` and `output_dimension` as parameters.